src/adapters/keystrokes_adapter.js

```diff
@@ -0,0 +1,28 @@
+import webChannel from '../lib/web_channel';
+import LilMacResults from '../collections/lil_mac_results';
+
+class KeystrokesAdapter {
+  constructor () {
+    this.results = new LilMacResults();
+    webChannel.on('printable-key', data => {
+      this.search(data.query);
```

Comment:
We should probably debounce this request. I think I used around 150ms in the prototype.